lib/main.go

```diff
@@ -200,6 +200,38 @@ func (obj *Main) Exit(err error) {
 	obj.exit <- err // trigger an exit!
 }
 
+// createPrefix handles logic to create (tmp) prefix directory.
+func (obj *Main) createPrefix(hostname string) (string, bool, error) {
+	istmp := false
```

Comment:
vars should be in the form `isTmp`, however I don't think this is needed at all. Just log the prefix we make.

Reply:
Fine by me, I just tried to maintain feature parity :)